applications/crossbar/src/modules/cb_tasks.erl

```diff
@@ -284,6 +290,13 @@ put(Context) ->
             crossbar_util:response_400(<<"bad request">>, Reason, Context)
     end.
 
+-spec task_account_id(cb_context:context()) -> api_ne_binary().
+task_account_id(Context) ->
+    case cb_context:account_id(Context) of
+        'undefined' -> cb_context:auth_account_id(Context);
```

Comment:
I don't see the link. Since you auth as superduper you have an account.
This root endpoint is not documented and furthermore no special authentication step ensures only superduper has rights to this endpoint (see kz_rates).
I'd say either fix these 2 points and remove is_allowed in kz_rates, or just use the normal endpoint.